.env.sample

```diff
@@ -0,0 +1,3 @@
+CLOUDINARY_URL=
+GMAIL_ADDRESS=
```

Comment:
do i need to add your gmail address and password to heroku config?

Reply:
think so, else you can make your own and change the files with my email inside into your email (there is a sender: <my email> somewhere)